bedrock/firefox/templates/firefox/whatsnew/beta/whatsnew-fx68.html

```diff
@@ -10,8 +10,7 @@
 {{ fxa_email_form(
     entrypoint='mozilla.org-whatsnew68beta',
     button_class='mzp-c-button',
-    utm_source='whatsnew',
-    utm_params={'campaign': 'fxa-embedded-form', 'content': 'whatsnew', 'medium': 'referral'})
```

Comment:
`utm_campaign` value on this form ends up being `-fxa-embedded-form` because we're omitting the value.